x-pack/solutions/observability/plugins/apm/server/routes/span_links/utils.ts

```diff
@@ -21,3 +22,27 @@ export function getBufferedTimerange({
     endWithBuffer: moment(end).add(bufferSize, 'days').valueOf(),
   };
 }
+
+export function mapOtelToSpanLink(otelSpanLinks: Partial<RawOtelSpanLink> | undefined): SpanLink[] {
+  const spanLink: SpanLink[] = [];
```

Comment:
Nit Maybe we can use plural here because it's an array? (same for `mapOtelToSpanLink`)
```suggestion
  const spanLinks: SpanLink[] = [];
```